searchcore/src/vespa/searchcore/proton/attribute/attribute_initializer.cpp

```diff
@@ -206,30 +192,27 @@ AttributeInitializer::loadAttribute(const AttributeVectorSP &attr,
 }
 
 void
-AttributeInitializer::setupEmptyAttribute(AttributeVectorSP &attr,
-                                          search::SerialNum serialNum,
+AttributeInitializer::setupEmptyAttribute(AttributeVectorSP &attr, search::SerialNum serialNum,
                                           const AttributeHeader &header) const
 {
     if (header.getCreateSerialNum() > _currentSerialNum) {
-        logAttributeTooNew(attr, header, _currentSerialNum);
+        logAttributeTooNew(header, _currentSerialNum);
     }
     if (serialNum < _currentSerialNum) {
-        logAttributeTooOld(attr, serialNum, _currentSerialNum);
+        logAttributeTooOld(header, serialNum, _currentSerialNum);
     }
     if (!headerTypeOK(header, attr->getConfig())) {
         logAttributeWrongType(attr, header);
     }
-    LOG(info, "Returning empty attribute vector for '%s'",
-            attr->getBaseFileName().c_str());
+    LOG(info, "Returning empty attribute vector for '%s'", attr->getName().c_str());
```

Comment:
Too little information (document type and document subdb name can no longer be deduced from log message).

Reply:
I pulled this one back. It will be redone later on.